docs/project-model/project-json-reference.md

```diff
@@ -0,0 +1,494 @@
+
+<a name="content"></a>
+## content
+Type: Globbing String
+
+A globbing string that specifies what files are considerd content by the build.
+
+<a name="resource"></a>
+## resource
+Type: Globbing String
+
+A globbing string that specifies what files are resource files.
+
+<a name="preprocess"></a>
+## preprocess
+Type: Globbing String
+
+A globbing string that specifies what files are included in preprocessing.
+
+<a name="publishExclude"></a>
+## publishExclude
+Type: Globbing String
+
+A globbing string that specifies what files are excluded from publishing.
+
+<a name="shared"></a>
+## shared
+Type: Globbing String
+
+A globbing string that specifies what files are shared, this is used for library export.
+
+<a name="namedResource"></a>
+## namedResource
+Type: Globbing String
+
+an object whos keys are names of named resources and whos values are the paths to each resource.
+
+<a name="packInclude"></a>
+## packInclude
+Type: Globbing String
+
+A globbing string that specifies what files are included when creating the nuget package.
+
+<a name="exclude"></a>
+## exclude
+Type: Globbing String
+
+A globbing string that specifies what files to exclude from the build.
+
+<a name="contentBuiltIn"></a>
+## contentBuiltIn
+Type: Globbing String
+
+A globbing string that specifies what files are considerd content by the build.
+
+<a name="compileBuiltIn"></a>
+## compileBuiltIn
+Type: Globbing String
+
+A globbing string that specifies what files are included in compilation.
+
+<a name="resourceBuiltIn"></a>
+## resourceBuiltIn
+Type: Globbing String
+
+A globbing string that specifies what files are resource files.
+
+<a name="excludeBuiltIn"></a>
+## excludeBuiltIn
+Type: Globbing String
+
+A globbing string that specifies what files to exclude from the build.
+
+<a name="dependencies"></a>
+## dependencies
+Type: Object
+
+An object that defines the package dependencies of the project, each key of this object is the name of a package and each value contains versioning information.
+
+    "dependencies": {
+        "System.Reflection.Metadata": "1.2.0-rc3-23811",
+        "Microsoft.Extensions.JsonParser.Sources": {
+          "type": "build",
+          "version": "1.0.0-rc2-16453"
+        },
+        "Microsoft.Extensions.HashCodeCombiner.Sources": {
+          "type": "build",
+          "version": "1.0.0-rc2-16054"
+        },
+        "Microsoft.Extensions.DependencyModel": "1.0.0-*"
+    }
+
+<a name="tools"></a>
+## tools
+Type: Object
+
+An object that defines package dependencies that are used as tools for the current project, not as references. Packages defined here are available in scripts that run during the build process, but they are not accessible to the code in the project itself. Tools can for exampel include code generators or post build tools that perform tasks related to packing.
+
+<a name="commands"></a>
+## commands
+Type: Object
+
+Commands are depricated in the cli. (?)
```

Comment:
Heh, unfortunately I can't answer that.  We can leave this as a question mark or mark it with a **TODO** for now.